plugins/docker/src/tasks/push.ts

```diff
@@ -60,5 +70,10 @@ export default class DockerPush extends Task<{
         }
       }
     }
+
+    // TODO either update "ci" types or add a new "docker" state??
+    writeState('ci', {
+      pushedImages
+    });
```

Comment:
We're not sure where we should be putting this info. We were originally thinking `ci` but maybe it makes more sense to add a new `docker` state?

Either way we'll need to update the types in `@dotcom-tool-kit/state`. Thoughts @apaleslimghost / @ivomurrell ?

Reply:
state is kind of a mess tbh, so i wouldn't worry too much about whether this is the "right" bit of state to put it in. `ci` is fine.